wormhole-connect/src/utils/wallet/solana.ts

```diff
@@ -178,25 +202,61 @@ async function createPriorityFeeInstructions(
   transaction: Transaction | VersionedTransaction,
   commitment?: Commitment,
 ) {
-  const response = await (isVersionedTransaction(transaction)
-    ? connection.simulateTransaction(transaction, {
-        commitment,
-        replaceRecentBlockhash: true,
-      })
-    : connection.simulateTransaction(transaction));
-  if (response.value.err) {
-    throw new Error(`Simulation failed: ${response.value.err}`);
+  if (isVersionedTransaction(transaction)) {
+    // This is required for versioned transactions - simulateTransaction throws
+    // if recentBlockhash is an empty string.
+    const { blockhash } = await connection.getLatestBlockhash(commitment);
+    transaction.message.recentBlockhash = blockhash;
```

Comment:
Can we set this using the `blockhash` in `signAndSendTransaction` instead to reduce RPC calls?

Reply:
I don't think so because the simulation fails if it's not set, and we have to do the simulation to create the priority fee ix before we call `signAndSend`